crates/containerd-shim-wasmtime/src/executors/container.rs

```diff
@@ -1,60 +1,19 @@
 use containerd_shim_wasm::sandbox::oci;
-use libcontainer::workload::{Executor, ExecutorError, EMPTY};
-use nix::unistd::{dup, dup2};
+use libcontainer::workload::default::DefaultExecutor;
+use libcontainer::workload::{Executor, ExecutorError};
 
-use std::ffi::CString;
-use std::io::Read;
-use std::{fs::OpenOptions, os::fd::RawFd, path::PathBuf};
-
-use libc::{STDERR_FILENO, STDIN_FILENO, STDOUT_FILENO};
-use nix::unistd;
 use oci_spec::runtime::Spec;
-
-const EXECUTOR_NAME: &str = "default";
+use std::io::Read;
+use std::{fs::OpenOptions, path::PathBuf};
 
 #[derive(Default)]
 pub struct LinuxContainerExecutor {
-    pub stdin: Option<RawFd>,
-    pub stdout: Option<RawFd>,
-    pub stderr: Option<RawFd>,
+    pub default_executor: DefaultExecutor,
```

Comment:
I think this could be private since we are handling it? then the shim wouldn't need to know about the default at all?